kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -15,11 +15,8 @@ enum class CompareOrder {
 /**
  * Compares two json trees, returning a detailed error message if they differ.
  */
-fun compare(expected: JsonNode, actual: JsonNode, mode: CompareMode, order: CompareOrder) =
-   compare(emptyList(), expected, actual, mode, order)
-
 fun compare(
-   path: List<String>,
+   path: List<String> = emptyList(),
```

Comment:
not sure we need the default here if it's only called internally.

Reply:
It would be a breaking change to not have the default since I removed the overload, but I can remove it if you'd like!

Reply:
This is brand new in 4.4 and can be marked internal anyway, so please add internal and feel free to break.